Assets/Shopify/UIToolkit/Shops/Generic/Classes/CartItemView.cs

```diff
@@ -0,0 +1,67 @@
+namespace Shopify.UIToolkit {
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System;
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    /// <summary>
+    /// A View Model for converting the CartItem data into a format for the CartItemView component.
+    /// </summary>
+    public struct CartItemViewModel {
+        public readonly string Title;
+        public readonly string VariantDescription;
+        public readonly string PriceText;
+        public readonly string QuantityText;
+        public readonly string ImageURL;
+
+        public CartItemViewModel(CartItem cartItem) {
+            var variant = cartItem.Variant;
+            var product = variant.product();
+
+            Title = product.title();
+            VariantDescription = variant.title();
+            PriceText = string.Format("${0}", variant.price());
+            QuantityText = cartItem.Quantity.ToString();
+
+            string imageURL;
+            try {
+                imageURL = variant.image().src();
```

Comment:
are we supposed to ignore this src attribute now? whats the deal with the deprecation

Reply:
Not sure - I'll investigate after this PR

Reply:
Should use transformedSrc now IIRC